tests/DropdownTest.php

```diff
@@ -119,14 +119,13 @@ public function testActive()
                     [
                         'label' => 'Dropdown2',
                         'items' => [
-                            ['label' => 'Page3', 'content' => 'Page3', 'url' => '/', 'active' => true],
+                            ['label' => 'Page3', 'content' => 'Page3', 'url' => '/', 'linkOptions' => ['class' => 'active']],
```

Comment:
If you keep `active` in `Dropdown`, you should keep your version here with `'active' => true`

Reply:
Absolutely correct! This one slipped through from copy